src/buffer/lru_k_replacer.cpp

```diff
@@ -14,6 +14,22 @@
 
 namespace bustub {
 
+LRUKNode::LRUKNode(size_t k, frame_id_t fid) : k_(k), fid_(fid) {}
+
+void LRUKNode::SetEvictable(bool evictable) {}
+
+auto LRUKNode::IsEvictable() -> bool { return false; }
+
+auto LRUKNode::GetFid() -> frame_id_t { return 0; }
```

Comment:
prefer throw NotImplementedError, otherwise students might forget to implement these functions.